gammapy/irf/psf/kernel.py

```diff
@@ -23,31 +23,30 @@ class PSFKernel:
     --------
     ::
 
-        import numpy as np
         from gammapy.maps import Map, WcsGeom, MapAxis
-        from gammapy.irf import EnergyDependentMultiGaussPSF, PSFKernel
+        from gammapy.irf import PSFMap
```

Comment:
Generally, the status of code examples in docstrings is not clear. I suppose for this type of lower level API this might still be useful since we probably don't explain much about it in the docs.

Reply:
Yes, I agree those are useful examples...I think we will be able to test them soon?